src/commands/secrets/download.ts

```diff
@@ -13,13 +15,15 @@ export default class SecretsDownload extends BaseCommand {
   static description = 'Download secrets from an account or an environment';
   static aliases = ['secrets', 'secrets/get'];
   static examples = [
-    'architect secrets ./mysecrets.yml',
-    'architect secrets --account=myaccount --environment=myenvironment ../mysecrets.yml',
+    'architect secrets --account=myaccount ./mysecrets.yml',
+    'architect secrets --account=myaccount --platform=myplatform ./mysecrets.yml',
+    'architect secrets --account=myaccount --environment=myenvironment ./mysecrets.yml',
   ];
   static flags = {
     ...BaseCommand.flags,
     ...AccountUtils.flags,
     ...EnvironmentUtils.flags,
+    ...PlatformUtils.flags,
```

Comment:
These need to be updated. We no longer want to pull them in like this since if you set the env of the process then those flags can be set implicity. We want to copy them over here instead and remove the `environment` field.